### ISSUE TITLE
Pages don't change on windows phone

### ISSUE THREAD

**Issue by luisrudge** [reapp/reapp#35]
The url changes but the pages don't. I tested the kitchen sink demo

**Comment by luisrudge** [comment]
The HN demo keeps the loading icon forever

**Comment by leeDav** [comment]
Confirmed on Nokia Lumia 635.

Hacker News indeed keeps the loading icon.
Kitchen Sink loads data and seems to change page, but it doesn't let me scroll below the initial fold.

**Comment by natew** [comment]
@leeDav I'm purchasing a windows phone today to start testing it on that platform. Thanks for the bug reports.

I did just push an update 20 minutes ago to kitchen sink, but nonetheless I'm excited to be able to actually test on real hardware.

**Comment by luisrudge** [comment]
Yeah. Now the page and scroll are working, but it's very very slow :(

**Comment by superghostmaster** [comment]
Does anyone happen to know what the JS implementation on Win Phone uses?  Is it effectively the same as native IE, or unique to the phone, etc.?
